src/python/grapl-http-service/mypy.ini

```diff
@@ -0,0 +1,13 @@
+[mypy]
+allow_redefinition = True
+disallow_any_generics = True
+disallow_incomplete_defs = True
+disallow_untyped_defs = True
+no_implicit_optional = True
+pretty = True
+show_error_context = True
+strict_equality = True
+warn_return_any = True
+warn_unreachable = True
+warn_unused_ignores = True
+
```

Comment:
It would be better to fold this into the main configuration file: https://github.com/grapl-security/grapl/blob/main/src/python/mypy.ini